app/serializers/webhook/article_serializer.rb

```diff
@@ -0,0 +1,11 @@
+module Webhook
+  class ArticleSerializer
+    include FastJsonapi::ObjectSerializer
+    set_type :article
+    belongs_to :user
```

Comment:
what about the organization?

Reply:
I haven't yet worked on the logic related to organizations, so I haven't included it yet.